libs/home/src/lib/home.component.html

```diff
@@ -10,30 +10,27 @@ <h1 class="logo-font">conduit</h1>
     <div class="row">
       <div class="col-md-9">
         <div class="feed-toggle">
-          <ul class="nav nav-pills outline-active">
+          <ul class="nav nav-pills outline-active" *ngIf="{ listConfig: listConfig$ | async } as model">
```

Comment:
I started using the new control flow. Could you use it also here?

If you do this, remove the CommonModule from the imports of the component and just import the AsyncPipe

Reply:
I don't think removal of CommonModule will be possible because AsyncPipe is placed in CommonModule. But usage of new control flow is nice - I pushed change for this.

Reply:
Hey @andrzejpindor . There is a standalone AsyncPipe you can import. Check my latest commit. I did the same for other components. I let the `home.component` for you

https://github.com/stefanoslig/angular-ngrx-nx-realworld-example-app/commit/bc75e6f71eaef42b21d1a6cc1115b09667a87321

Reply:
Nice! I didn't notice this earlier.